### PR TITLE
Added paginator

### PR DESCRIPTION
**Pull request by kaspian-xz**
Issue: https://github.com/odessajavaclub/odessajavaclub.github.io/issues/2

image

### CODE REVIEW

**Review by apletnev — APPROVED** [review]
Nice, thank you

File: Gemfile [hunk 1]
```diff
@@ -21,6 +21,7 @@ gem "jekyll-theme-modernist", "~> 0.1.1"
 # If you have any plugins, put them here!
 group :jekyll_plugins do
   gem "jekyll-feed", "~> 0.6"
+  gem "jekyll-paginate-v2"
```
Comment (msashkin) [inline Gemfile:24]: Should we constraint the minimum version of this plugin?

**Review by msashkin — APPROVED** [review]

File: Gemfile (hunk 1 above)
Comment (alexbevziuk-okta) [reply Gemfile]: @msashkin didn't find any mention about version in available examples or plugin doc

Comment (kaspian-xz) [reply Gemfile:24]: @msashkin didn't find any mention about version in available examples or plugin doc

**Pull request merged**